src/Paysera/WalletApi/Util/Router.php

```diff
@@ -9,7 +9,7 @@
 class Paysera_WalletApi_Util_Router
 {
     const ENDPOINT_API_PROD = 'https://wallet.paysera.com';
-    const ENDPOINT_AUTH_PROD = 'https://www.paysera.com/frontend';
+    const ENDPOINT_AUTH_PROD = 'https://bank.paysera.com/frontend';
```

Comment:
It looks like the `https://www.paysera.com/frontend` will redirect to `https://bank.paysera.com/frontend` automatically.

Reply:
Yes, you are right. This change is made to reduce redirects amount.